coverage/historical/github/historical_coverage_report.py

```diff
@@ -383,13 +388,36 @@ def get_pull_requests(token, workspace, repo_name):
               }}
             }}
             baseRefOid
+            comments_authors: comments(first: 100) {{
+              nodes {{
+                author {{
+                  login
+                }}
+              }}
+            }}
           }}
         }}
       }}
     }}
     """
-    result = run_query(query, token)
-    return result['data']['repository']['pullRequests']['nodes']
+    result = run_query(query_template.format(workspace=workspace, repo_name=repo_name, after=""), token)
+    nodes = result['data']['repository']['pullRequests']['nodes']
+    has_next_page = result['data']['repository']['pullRequests']['pageInfo']['hasNextPage']
+    end_cursor = result['data']['repository']['pullRequests']['pageInfo']['endCursor']
+    while has_next_page:
+        result = run_query(query_template.format(workspace=workspace, repo_name=repo_name, after=f', after: "{end_cursor}"'), token)
+        nodes += result['data']['repository']['pullRequests']['nodes']
+        has_next_page = result['data']['repository']['pullRequests']['pageInfo']['hasNextPage']
+        end_cursor = result['data']['repository']['pullRequests']['pageInfo']['endCursor']
+    for node in nodes:
+        comments_authors = node['comments_authors']['nodes']
+        for comment_author in comments_authors:
+            if comment_author['author']['login'] in ['vibinex-dpu', 'vibinex-code-review']:
```

Comment:
Bitbucket will have different names of apps, just noting, no change requested

Reply:
Yes, I think the approach will have to be very different there because the comment is added in the name of the installer person.